ui/elemental_shaman/sim.ts

```diff
@@ -234,6 +234,7 @@ export class ElementalShamanSimUI extends IndividualSimUI<Spec.SpecElementalSham
 				],
 				// Whether to include 'Execute Duration (%)' in the 'Encounter' section of the settings tab.
 				showExecuteProportion: false,
+				showHealthPicker: false,
```

Comment:
This option shouldn't be something that's specific to each individual sim. Remove the option and instead check for isIndividualSim in the encounter_picker.ts